common/errors/errors.go

```diff
@@ -30,6 +30,11 @@ func (err *Error) Unwrap() error {
 	return err.goerror.Err
 }
 
+// Goerr returns goerrors.Error
+func (err *Error) Goerr() *goerrors.Error {
```

Comment:
https://github.com/pastelnetwork/gonode/tree/master/common#errors

> Under the hood, the errors package is using the go-errors library, but this may change in the future, so the rest of the code should not depend on go-errors directly.

Unit test should test our package instead of `goerror`. Please don't refer to `goerror` at all.

Reply:
@Levko errors package is  heavily tied to goerrors
mysuggestion would be to change the struct to accept error interface instead of goerrors.Error
that's why i needed to expose Goerr() to get underlying info

Reply:
I added Stack() to avoid exposing Goer(), so that we can change it to interface later.

Reply:
> @Levko errors package is heavily tied to goerrors
> mysuggestion would be to change the struct to accept error interface instead of goerrors.Error
> that's why i needed to expose Goerr() to get underlying info

You should use only those method which are used other packages, otherwise you test method that you wrote for unit_test, no sense. Why don't you use `Unwrap()` or compare string from `Error()`. Anyway just image that we don't use `goerrors`, and just pretend that you don't know what the Error struct contains in inside, pretend you know as much as the packages that use it.

> the struct to accept error interface instead of goerrors.Error
if you know better solution, please write your code, if it's really will be better than we have now, I will say you many thank you.